srv-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: '3'
+
```

Comment:
This is the same as the block in ``docker-compose``
Not possible to have a common block that could be re-used